Simplified some intro language for email macros guide (#2347)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

> **Note:** Some templates listed in the [variables tables](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-expression-language) may not appear in your org. To obtain these templates, contact [Okta Support](https://support.okta.com/help/s/?_ga=2.17747641.1660906902.1597076228-1076744453.1575496867).

### Enhanced Email Macros
<ApiLifecycle access="ea" />

#### Overview of what this EA feature enables

> The Early Access (EA) Enhanced Email Macros feature provides you with the functionality to customize the email templates with even greater flexibility than EL. It changes the underlying email templating engine from EL variables to [Velocity Templating Language (VTL)](https://velocity.apache.org/). This allows y so that you can use enhanced conditional logic, all of the attributes in the Okta [User Profile object](/docs/reference/api/users/#profile-object), as well as some of the org attributes in these macros as documented below.

Reviewer: ```suggestion
The Early Access (EA) Enhanced Email Macros feature provides you with the functionality to customize the email templates with even greater flexibility than EL. It changes the underlying email templating engine from EL variables to [Velocity Templating Language (VTL)](https://velocity.apache.org/). This allows you to customize your org's email templates so that you can use enhanced conditional logic, all of the attributes in the Okta [User Profile object](/docs/reference/api/users/#profile-object), and some of the org attributes in these macros, as documented below.
```

##### EA feature requirements

> To access this self-serviced EA feature, you need to enable `ENHANCED_EMAIL_MACROS`in the feature flag. For free orgs, you will need to first contact Okta support (`support@okta.com`) to request email customization.

Reviewer 2: Jakub said we never documented the actual feature flag in our docs. If the end user admin can set it maybe send them to this Okta link: https://help.okta.com/en/prod/Content/Topics/Security/manage-EA-and-beta-features.htm or simply mention to contact Okta support to enable the feature.

Author: Thanks, there's 2 FFs mentioned here. ENHANCED_EMAIL_MACROS which is full self-service EA via Admin Console, and the email customization flag which we cannot name and must be turned on by support after Security team evaluates the request for risk.
